StoreKitUnitTests/StoreKitConfigTestCase.swift

```diff
@@ -36,7 +38,7 @@ class StoreKitConfigTestCase: XCTestCase {
         testSession.disableDialogs = true
         testSession.clearTransactions()
 
-        self.waitForStoreKitTestIfNeeded()
+//        self.waitForStoreKitTestIfNeeded()
```

Comment:
If this approach works I'll remove this whole thing
(And let's not forget to remove the environment variable).